recipes/simplification/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "simplification" %}
+{% set version = "0.5.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/urschrei/{{ name }}/tarball/{{ version }}
+  sha256: 1ab55d160719803cecb79d6cfc6c4a488011821d49224fbe3acb08b986b2298a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37, py>38]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip >=8.1.2
+  run:
+    - python
+
+
+test:
+  imports:
+    - simplification
+    - simplification.test
+
+about:
+  home: https://github.com/urschrei/simplification
+  license: MIT
+  license_family: MIT
+  license_file: license.txt
+  summary: 'A LineString using the Ramer–Douglas–Peucker or Visvalingam-Whyatt algorithms'
+  doc_url: https://simplejson.readthedocs.io/
```

Comment:
Left-over from template? Can be removed if there's nothing relevant?

Reply:
thanks, fixing.